src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -292,9 +292,11 @@ public void addInformation(ItemStack stack, EntityPlayer player, List list, bool
 		addStringToTooltip(StatCollector.translateToLocal("botaniamisc.lokiDescription"), list);
 		addStringToTooltip(StatCollector.translateToLocal("botaniamisc.lokiDescription2"), list);
 		addStringToTooltip("", list);
-		addStringToTooltip(EnumChatFormatting.WHITE +StatCollector.translateToLocal("botaniamisc.lokiCurrent"), list);		
-		addStringToTooltip(StatCollector.translateToLocal("botaniamisc.lokiState") + ": " + getOnOffString(isRingEnabled(stack)), list);
-		addStringToTooltip(StatCollector.translateToLocal("botaniamisc.breaking") + ": " + getOnOffString(isRingBreakingEnabled(stack)), list);
+		addStringToTooltip(EnumChatFormatting.WHITE +StatCollector.translateToLocal("botaniamisc.lokiCurrent"), list);
+		if (stack.hasTagCompound()) {
+			addStringToTooltip(StatCollector.translateToLocal("botaniamisc.lokiState") + ": " + getOnOffString(isRingEnabled(stack)), list);
+			addStringToTooltip(StatCollector.translateToLocal("botaniamisc.breaking") + ": " + getOnOffString(isRingBreakingEnabled(stack)), list);
+		}
```

Comment:
Instead of adding this condition here I suggest to modify methods `isRingEnabled` and `isRingBreakingEnabled`. This will prevent other possible issues caused by absent NBT.

Reply:
Agreed. I like that change.